src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from '@/components/CalculateCurrentStock';
+export * from '@/components/ForestManagementProvider';
+export * from '@/components/Forestry';
+export * from '@/components/OperationClass';
+export * from '@/components/OperationClasses';
+
+export type { default as FeasibilityStudyConfig } from '@/models/FeasibilityStudyConfig';
```

Comment:
I think we should also export the yield tables since we need to be able to select them in the form in the app?

Reply:
Done